example/src/main.rs

```diff
@@ -0,0 +1,61 @@
+#![recursion_limit="128"]
+extern crate aws_lambda;
+#[macro_use] extern crate error_chain;
+#[macro_use] extern crate log;
+
+use aws_lambda::event::s3::S3Event;
+
+pub mod errors {
+    use std;
+
+    // Create the Error, ErrorKind, ResultExt, and Result types
+    error_chain! {
```

Comment:
Yeah, I can probably leave it as a comment in the Cargo.toml for error-chain users, since you can't use error-chain default features in musl without a lot of wrangling.